app/src/main/java/com/kickstarter/viewmodels/ProjectNotificationSettingsViewModel.java

```diff
@@ -8,39 +8,47 @@
 import com.kickstarter.models.ProjectNotification;
 import com.kickstarter.services.ApiClientType;
 import com.kickstarter.ui.activities.ProjectNotificationSettingsActivity;
-import com.kickstarter.viewmodels.errors.ProjectNotificationSettingsViewModelErrors;
-import com.kickstarter.viewmodels.outputs.ProjectNotificationSettingsViewModelOutputs;
 
 import java.util.List;
 
 import rx.Observable;
 import rx.subjects.PublishSubject;
 
-public final class ProjectNotificationSettingsViewModel extends ActivityViewModel<ProjectNotificationSettingsActivity> implements
-  ProjectNotificationSettingsViewModelOutputs, ProjectNotificationSettingsViewModelErrors {
+public interface ProjectNotificationSettingsViewModel {
 
-  public ProjectNotificationSettingsViewModel(final @NonNull Environment environment) {
-    super(environment);
-
-    final ApiClientType client = environment.apiClient();
+  interface Outputs {
+    Observable<List<ProjectNotification>> projectNotifications();
+  }
 
-    this.projectNotifications = client.fetchProjectNotifications()
-      .compose(Transformers.pipeErrorsTo(this.unableToFetchProjectNotificationsError));
+  interface Errors {
```

Comment:
aha, so we're also moving away from having `Errors` as a separate interface, you can move this Observable into `Outputs` since this error is technically just another output for the UI!

Reply:
Oh I see 👍 I will make that change! :)